client/components/popover/style.scss

```diff
@@ -208,7 +208,7 @@
 	// with gridicons
 	.gridicon {
 		color: lighten( $gray, 10 );
-		vertical-align: bottom;
+		vertical-align: middle;
```

Comment:
Hmm. I think `bottom` works for any of the "circle" icons like the history icon. I think the download icon will need to be positioned itself to work and I would do it within this component. Something like:

```
.gridicons-cloud-download {
  position: relative;
  top: 2px;
}
```